.github/workflows/ci.yml

```diff
@@ -9,7 +9,7 @@ on:
 jobs:
   CI:
     runs-on: ubuntu-latest
-    container: heliumsystems/builder-erlang:1
+    container: heliumsystems/builder-erlang:2
```

Comment:
Looking at this guy's image layers in dockerhub i'm not sure which repo this is built from but it looks like it could use some updating as well if I can be pointed in its direction

Reply:
we need to get rid of that.. no more builder-erlang